testing/src/main/java/org/oppia/android/testing/platformparameter/TestPlatformParameterModule.kt

```diff
@@ -354,12 +343,6 @@ class TestPlatformParameterModule {
       enablePerformanceMetricsCollection = value
     }
 
-    /** Enables forcing [EnableContinueButtonAnimation] platform parameter flag from tests. */
-    @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    fun forceEnableContinueButtonAnimation(value: Boolean) {
```

Comment:
You need to find and remove usages of this method. I can see that there are dagger build failures in your tests. See [stacktrace](https://github.com/oppia/oppia-android/actions/runs/6878856304/job/18741659797?pr=5228) and [wiki debugging help](https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results).

Reply:
Ok, on it

Reply:
Hi @adhiamboperes , I think I need more clarification or help, I can't find extra usages of this method in the code. And I have looked through the stacktrace as well.

Reply:
This is used in multiple test files. You can use Ctrl+Shift+F to do a global search for the method, or use Ctrl+B/Ctrl+Click on the method name.

Reply:
Thanks, this helped @adhiamboperes